ansys/mapdl/core/convert.py

```diff
@@ -514,7 +514,12 @@ def store_command(self, function, parameters):
             elif "ARG" in parameter and self._infunction:
                 parsed_parameters.append("%s" % parameter)
             else:
-                parsed_parameters.append('"%s"' % parameter)
+                # Removing strings '' and "" because they are going to be added by the converter module.
+                if parameter.startswith("'") and parameter.endswith("'"):
+                    parameter = parameter[1:-1]
+                if parameter.startswith('"') and parameter.endswith('"'):
+                    parameter = parameter[1:-1]
+                parsed_parameters.append('"%s"' % parameter)#
```

Comment:
Consider f-strings in the future:
```suggestion
                parsed_parameters.append(f'"{parameter}"')
```